src/Integration.Vsix/Commands/PackageCommandManager.cs

```diff
@@ -44,7 +46,8 @@ public PackageCommandManager(IMenuCommandService menuService)
         public void Initialize(ITeamExplorerController teamExplorerController,
             IProjectPropertyManager projectPropertyManager,
             IProjectToLanguageMapper projectToLanguageMapper,
-            IOutputWindowService outputWindowService)
+            IOutputWindowService outputWindowService,
+            IBrowserService browserService)
```

Comment:
I don't like that this class gets the dependencies of all the commands it initializes.
We might need to revisit it later, once we have more use cases.

Reply:
Yea I agree, its starting to get ugly